Chessington.GameEngine/Board.cs

```diff
@@ -25,37 +27,32 @@ public void AddPiece(Square square, Piece pawn)
         {
             board[square.Row, square.Col] = pawn;
         }
-    
+
         public Piece GetPiece(Square square)
         {
             return board[square.Row, square.Col];
         }
-        
+
         public Square FindPiece(Piece piece)
         {
             for (var row = 0; row < GameSettings.BoardSize; row++)
-                for (var col = 0; col < GameSettings.BoardSize; col++)
-                    if (board[row, col] == piece)
-                        return Square.At(row, col);
+            for (var col = 0; col < GameSettings.BoardSize; col++)
+                if (board[row, col] == piece)
+                    return Square.At(row, col);
 
             throw new ArgumentException("The supplied piece is not on the board.", "piece");
         }
 
         public void MovePiece(Square from, Square to)
         {
             var movingPiece = board[from.Row, from.Col];
-            if (movingPiece == null) { return; }
+            if (movingPiece == null) return;
```

Comment:
This is nitpicky, but usually it's preferred to use brackets, e.g.
if (movingPiece == null)
{
    return;
}